pkg/config/redisstore.go

```diff
@@ -18,12 +18,36 @@ import (
 	"fmt"
 	"net/url"
 	"strconv"
+	"strings"
+	"sync"
+	"time"
 
 	"github.com/golang/glog"
+	"github.com/mediocregopher/radix.v2/pubsub"
 	"github.com/mediocregopher/radix.v2/redis"
 )
 
+const (
```

Comment:
I think this whole file should move into `pkg/config/redis`, `package redis`. Then the names of essentially all the functions here can be simplified:

`config.doesRedisConfigSupportsChangeNotifications` -> `redis.supportsChangeNotifications`
`config.setupRedisConnection` -> `redis.NewConnection`